cpp/oneapi/dal/algo/pca/backend/gpu/train_kernel_cov_impl_dpc.cpp

```diff
@@ -70,7 +70,7 @@ result_t train_kernel_cov_impl<Float>::operator()(const descriptor_t& desc, cons
     const auto data_nd = pr::table2ndarray<Float>(q_, data, alloc::device);
 
     auto [sums, sums_event] = compute_sums(q_, data_nd);
-
+    sums_event.wait_and_throw();
```

Comment:
Just for my understanding: Why we adding these blocking waits, making this async?

Reply:
we observe strange events control work when we use syevd function from oneMKL, these wait_and_throw functions are a workaround